File: In C/linkedList.c
#include <stdio.h>
#include <stdlib.h>

struct Node
{
    int data;
    struct Node *next;
};

int main()
{
    struct Node *head, *second, *third;
    head = (struct Node *)malloc(sizeof(struct Node));
    second = (struct Node *)malloc(sizeof(struct Node));
    third = (struct Node *)malloc(sizeof(struct Node));

    head->data = 1;
    head->next = second;
    second->data = 2;
    second->next = third;
    third->data = 3;
    third->next = NULL;

    // printing the linkedlist
    while (head != NULL)
    {
        printf("Element is :%d\n", head->data);
        head = head->next;
    }

    return 0;
}

File: In Java/src/dsa/linkedList.java
package dsa;

class Node{
	Node next;
	int value;
	public Node(int d) {
		this.value=d;
		this.next=null;
	}
}

class LinkedList23{
	Node head;
	public void create_linkedlist() {
		Node first=new Node(1);
		Node second=new Node(2);
		Node third=new Node(3);
		
		head=first;
		first.next=second;
		second.next=third;
	}
	
	public void printList() {
		while(head!=null) {
			System.out.print(head.value+"->");
			head=head.next;
		}
		System.out.print("null");
	}
	
}



public class linkedList {

	public static void main(String[] args) {
		LinkedList23 ll=new LinkedList23();
		ll.create_linkedlist();
		ll.printList();

	}

}
